libs/base/Language/Reflection/TTImp.idr

```diff
@@ -209,22 +248,34 @@ mutual
                    (opts : List FnOpt) ->
                    (type : ITy) ->
                    IClaimData
-
+  ||| A top-level declaration
   public export
   data Decl : Type where
-       IClaim : WithFC IClaimData -> Decl
-       IData : FC -> WithDefault Visibility Private -> Maybe TotalReq -> Data -> Decl
-       IDef : FC -> Name -> (cls : List Clause) -> Decl
+       ||| A type ascription, `a : b`.
+       ||| Called a claim because of Curry Howard, the statement `x : p` is equivalent to `x` is a proof of `p`.
```

Comment:
I'm not sure this line is a valid one. I thought it is a claim because it starts definition of a function, but does not define its body